app/src/main/res/layout/organiser_fragment_profile.xml

```diff
@@ -43,203 +44,339 @@
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.492"
+                app:layout_constraintStart_toStartOf="@+id/O_profile_image"
+                app:layout_constraintTop_toTopOf="@+id/O_profile_image"
+                app:layout_constraintVertical_bias="0.942"
+                app:srcCompat="@drawable/e_p_ic_edit"
+                tools:srcCompat="@drawable/e_p_ic_edit" />
 
-    <TextView
-        android:id="@+id/E_P_email"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="12dp"
-        android:fontFamily="@font/roboto_mono_medium"
-        android:text="Email:"
-        android:textSize="20sp"
-        android:textColor="@color/dark_purple"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.17"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/E_P_textPronouns" />
-
-    <EditText
-        android:id="@+id/E_P_textEmail"
-        android:layout_width="wrap_content"
-        android:layout_height="48dp"
-        android:ems="13"
-        android:focusable="false"
-        android:focusableInTouchMode="false"
-        android:fontFamily="@font/roboto_mono_medium"
-        android:hint="moodeng@ualberta.ca"
+            <Button
+                android:id="@+id/O_P_uploadImage"
+                android:layout_width="wrap_content"
+                android:layout_height="48dp"
+                android:layout_marginStart="132dp"
+                android:layout_marginTop="24dp"
+                android:backgroundTint="@color/dark_purple"
+                android:text="Upload Image"
+                android:textColor="@color/light_purple"
+                android:visibility="gone"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_profile_image" />
 
-        android:inputType="textEmailAddress"
-        android:textColor="@color/dark_purple"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.503"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/E_P_email" />
+            <Button
+                android:id="@+id/O_P_deleteImage"
+                android:layout_width="wrap_content"
+                android:layout_height="48dp"
+                android:layout_marginStart="12dp"
+                android:layout_marginTop="24dp"
+                android:backgroundTint="@color/dark_purple"
+                android:text="Delete Image"
+                android:textColor="@color/light_purple"
+                android:visibility="gone"
+                app:layout_constraintStart_toEndOf="@+id/O_P_uploadImage"
+                app:layout_constraintTop_toBottomOf="@+id/O_profile_image" />
+
+            <TextView
```

Comment:
My initial impression about using O_P_name for the textVew and O_P_textName for the editText is that this convention isn't very intuitive. When I saw this I thought that the O_P_textName would have been the textView.

Reply:
Looks great